### PR TITLE
[Components][SeasonSelector] Update the years array to use objects as season data.

### PR DESCRIPTION
**Pull request by KenSparks-Dev**
[Components][SeasonSelector] Update the years array to use objects as season data. Update years map to accept object data from the years array. [styles][home.module.css] Update the style of displayed data for better rendering

### CODE REVIEW

File: pages/components/SeasonSelector.js [hunk 1]
```diff
@@ -1,18 +1,102 @@
 import React, { useState }from 'react';
 
 export default function SeasonSelector({onChange}) {
-    const years = ['Seasons', '2003', '2004', '2005', '2006']
+    const years = [
```
Comment (aisflat439) [inline pages/components/SeasonSelector.js]: Usually, in javascript, the convention is that a variable thats never gonna change (like seasons) are `UPPER_CASE`. So , I'd expect this as `SEASONS`.

Also, IMO, this should be in a separate file called constants.

File: pages/components/SeasonSelector.js [hunk 2]
```diff
@@ -1,18 +1,102 @@
...
       let season = event.target.value
       setChosenSeason(season)
       onChange(season)
   }
     return (
-        <select name="" id="" onChange={handleChangeSeason}>
-            {years.map((year) => {
-              return  <option key={year} value={year}>{year}</option>
+      <>
+      <span>Seasons: </span>
```
Comment (aisflat439) [inline pages/components/SeasonSelector.js]: semantic HTML would have me thinking this should be a label

**Review by aisflat439 — APPROVED** [review]
Looks good. Make those changes I asked for. Nice work

**Comment by KenSparks-Dev** [comment]
@aisflat439 Can this be merged?

**Comment by aisflat439** [comment]
Sure. It's your project always feel welcome to merge. IMO you should change years to `SEASONS` and put it in a constant file. then import it to this file.

**Pull request merged**